src/main/java/graphql/execution2/result/ExecutionResultMultiZipper.java

```diff
@@ -85,6 +87,7 @@ private ExecutionResultZipper moveUp(ExecutionResultNode parent, List<ExecutionR
     }
 
     private Map<ExecutionResultNode, List<ExecutionResultZipper>> zipperWithSameParent(List<ExecutionResultZipper> zippers) {
+        //todo: change to LinkedHashMap
         return zippers.stream().collect(Collectors.groupingBy(ExecutionResultZipper::getParent));
     }
```

Comment:
TODO ^

I think you put in a supplier of the Map

Reply:
done

Reply:
Yeah I reckon create its own helper method